app/models/container_template.rb

```diff
@@ -0,0 +1,15 @@
+class ContainerTemplate < ApplicationRecord
+  include CustomAttributeMixin
+
+  belongs_to :ext_management_system, :foreign_key => "ems_id"
```

Comment:
We do use the same for Container instead of connecting it through ContainerGroup, indeed it would be faster because it will be indexed.
Using `includes` when ever loading the projects will also work, I'm good with either way.